crates/bevy_sprite/src/render/mod.rs

```diff
@@ -396,7 +396,17 @@ pub fn extract_sprites(
             continue;
         }
         if let Some(texture_atlas) = texture_atlases.get(texture_atlas_handle) {
-            let rect = Some(texture_atlas.textures[atlas_sprite.index]);
+            let rect = Some(
+                *texture_atlas
+                    .textures
+                    .get(atlas_sprite.index)
+                    .unwrap_or_else(|| {
+                        panic!(
+                            "Sprite index {} does not exist for texture atlas.",
```

Comment:
Would be nice if this also contained the path to the atlas, but I'm not sure how easy it would be to get it from here

Reply:
Probably could provide information about the handle instead, better than nothing.

Reply:
added the handle id. The error message is now
```text
'Sprite index 2000 does not exist for texture atlas handle Id(7233c597-ccfa-411f-bd59-9af349432ada, 15037242661771820763).'
```
Not sure if the info is actually useful or now.